deploy/openj9-chart/values.yaml

```diff
@@ -0,0 +1,39 @@
+###############################################################################
+# Copyright (c) 2019,2020 IBM Corp.
```

Comment:
```suggestion
# Copyright (c) 2019,2020 IBM Corp.
```

2019?  Should these all be `2020, 2020`?

Reply:
Thanks, all of the copyright date has been fixed in latest commit.